src/components/PageElements/PageElements.tsx

```diff
@@ -106,7 +106,11 @@ const PageElements: React.FC<PageElementProps> = ({
 
           return (
             <div key={`question_${element.id}`}>
-              <Segment style={inlineStyles(isChangeRequest, isSummary)}>
+              <Segment
+                basic
+                className="summary-page-element"
+                style={inlineStyles(isChangeRequest, isSummary)}
```

Comment:
I guess you'll remove that in the change request branch

Reply:
Yeah, that's the plan. I think that these dynamic borders might need to be inline styles -- it seems like the simplest way, but we can address that when we get to these.